src/coord/src/catalog/builtin.rs

```diff
@@ -253,6 +253,18 @@ lazy_static! {
         id: GlobalId::System(2013),
         index_id: GlobalId::System(2014),
     };
+    pub static ref MZ_COLUMNS: BuiltinTable = BuiltinTable {
+        name: "mz_columns",
+        desc: RelationDesc::empty()
+            .with_column("qualified_name", ScalarType::String.nullable(false))
+            .with_column("global_id", ScalarType::String.nullable(false))
+            .with_column("field_number", ScalarType::String.nullable(false))
+            .with_column("field", ScalarType::String.nullable(false))
+            .with_column("nullable", ScalarType::String.nullable(false))
```

Comment:
Can you make this a `ScalarType::Bool`? Using the strings "YES" or "NO" is a weird MySQL-ism that probably dates from the days before SQL had a boolean type. The query that powers `SHOW INDEXES` should use something like `CASE nullable WHEN true THEN 'YES' ELSE 'NO' END` to recover the silly MySQL behavior—but this way folks using the latest-n-greatest `mz_catalog` stuff will get modern inventions like boolean types.